hack/make.sh

```diff
@@ -123,7 +123,7 @@ function target()
 			fi
 		done
 
-		pouch pull registry.hub.docker.com/library/busybox:latest >/dev/null
+		#pouch pull registry.hub.docker.com/library/busybox:latest >/dev/null
```

Comment:
I wish that we can lock down one usable image and in every test suite, we use this image. 
So we can remove `command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
}` in every test suite. Only when we need another image, we pull that image.

Although when pulling an existent image costs no time ideally, actually it will take pouch daemon about 2 or 3 second to process. If we remove that command `command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)}`, I think we could reduce maybe several minutes to finish the CI. 
@Letty5411 @HusterWan

Reply:
As we discussed before, this will impact the intact of a test suite. We could use API instead of Cli to cut the cost time.